### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Maxwell-Yang-PDFTron**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,795 @@
+# PDFTron Flutter API
+
+- [Utility Function APIs](#Utility-Function-APIs)
+- [Viewer Function APIs](#Viewer-Function-APIs)
+- [Event APIs](#Event-APIs)
+- [Viewer Configurations](#Viewer-Configurations)
+
+## Utility Function APIs
```
Comment (sgong-pdftron) [inline API.md:8]: i don't think it was called like this for React Native? maybe just "Methods"

File: API.md [hunk 2]
```diff
@@ -0,0 +1,795 @@
...
+key | String | true | your PDFTron license key
+
+Returns a Future.
+
+```dart
+PdftronFlutter.initialize('your_licensey_key');
+```
+
+
+## Viewer Function APIs
```
Comment (sgong-pdftron) [inline API.md:49]: "Viewer"

File: API.md [hunk 3]
```diff
@@ -0,0 +1,795 @@
...
+Closes all documents that are currently opened in a multiTab environment (that is, [multiTabEnabled](#multiTabEnabled) is true in the config).
+
+Returns a Future.
+
+```dart
+PdftronFlutter.closeAllTabs();
+```
+
+
+## Event APIs
```
Comment (sgong-pdftron) [inline API.md:462]: "Events"

File: API.md [hunk 4]
```diff
@@ -0,0 +1,795 @@
...
+#### startDocumentErrorListener
+Event is raised when the document has errors when loading.
+
+```dart
+var documentErrorCancel = startDocumentErrorListener((){
+  print("flutter document loaded unsuccessfully");
+});
+```
+
+### Viewer UI Configuration
```
Comment (sgong-pdftron) [inline API.md:492]: "Viewer"

File: API.md [hunk 5]
```diff
@@ -0,0 +1,795 @@
...
+var fieldChangedCancel = startFormFieldValueChangedListener((fields)
+{
+  for (Field field in fields) {
+    print("Field has name ${field.fieldName}");
+    print("Field has value ${field.fieldValue}");
+  }
+});
+```
+
+### Bookmark
```
Comment (sgong-pdftron) [inline API.md:605]: "Bookmarks"

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md (hunk 1 above)
Comment (Maxwell-Yang-PDFTron) [reply API.md]: For React Native, there is only RNPdftron and the viewer component. But here, I also wish to separate the static utility functions and the other viewer-related functions,

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**